### PR TITLE
Fix syntax error on postgres custom select

### LINKED ISSUE #348: Autocount is breaking on custom SELECTs
The method `JSONAPI::Resource#find_count` should be changed to:

``` ruby
 def self.find_count(filters, options = {})
  filter_records(filters, options).count(:all)
 end
```

so the SELECT is cleared. This is how `ActiveRecord::Relation#size` is implemented, BTW.

This problem emerges as a syntax error on any custom select, specially the ones you make just to please Postgres’s GROUP BY clauses.

### PR DESCRIPTION
**Pull request by lgebhardt**
Change count to count(:all)

fixes #348

### CODE REVIEW

File: lib/jsonapi/resource.rb [hunk 1]
```diff
@@ -483,8 +483,8 @@ def sort_records(records, order_options)
       end
 
       def find_count(filters, options = {})
-        filter_records(filters, options).count
-      end
+        filter_records(filters, options).count(:all)
+     end
```
Comment (dgeb) [inline lib/jsonapi/resource.rb]: did you drop a space here accidentally?

**Comment by dgeb** [comment]
Looks good otherwise :+1:

**Comment by lgebhardt** [comment]
Fixed

**Comment by dgeb** [comment]
Thanks! :shipit:

**Pull request merged**